tests/eclient/image/files/entrypoint.sh

```diff
@@ -19,8 +19,23 @@ rm -rf /run/*
 mkdir -p /run/sshd
 mkdir -p /run/nginx
 
+# eclient can be also used as a router app
+IP_FORWARDING="$(sysctl -n net.ipv4.ip_forward)"
+if [ "$IP_FORWARDING" -eq 1 ]; then
+    # With bare containers (eve.accel=false), IP forwarding is always enabled.
+    echo "IP forwarding is already enabled"
+else
+    echo "Enabling IP forwarding..."
+    sysctl -w net.ipv4.ip_forward=1 && echo "IP forwarding is now enabled"
```

Comment:
Will it work with non-accelerated EVE-OS (in case of running inside of container instead of container-in-VM)?

Reply:
See the comment `With bare containers...` above. In that case this does nothing.

Reply:
Ah, I see, thank you! Seems it come from EVE-OS kernel.